### PR TITLE
Use alt az

### LINKED ISSUE #109: fix azimuth interface with UVBeam
Currently when UVBeam is called, azimuth angle is not being converted from astropy frame to the UVBeam/CST/Radio Astronomy convention.

### PR DESCRIPTION
**Pull request by bhazelton**
fixes #109

### CODE REVIEW

File: pyuvsim/analyticbeam.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def peak_normalize(self):
 
     @profile
     def interp(self, az_array, za_array, freq_array):
-        # (Naxes_vec, Nspws, Nfeeds or Npols, freq_array.size, az_array.size)
+        # The azimuth here has the UVBeam convention: North of East(East=0, North=90)
```
Comment (dannyjacobs) [inline pyuvsim/analyticbeam.py]: az_array = azimuth in degrees from East rotating towards the North. East=0, North = 90
za_array = zenith angle in degrees away from zenith. straight up=0, horizon=90
freq_array = frequencies in Hz

Three arrays may be in arbitrary shape, but shapes of all three must be the same.

File: pyuvsim/source.py [hunk 2]
```diff
@@ -116,16 +119,16 @@ def coherency_calc(self, time, telescope_location):
...
-        calculate the azimuth & zenith angle for this source at a time & location
+        calculate the altitude & azimuth for this source at a time & location
 
         Args:
             time: astropy Time object
             telescope_location: astropy EarthLocation object
 
         Returns:
-            (azimuth, zenith_angle) in radians
+            (altitude, azimuth) in radians
```
Comment (dannyjacobs) [inline pyuvsim/source.py]: Looks like we switch between degrees and radians.  Should we try to be consistent?

File: pyuvsim/utils.py [hunk 3]
```diff
@@ -75,3 +76,65 @@ def cirs_to_tee_ra(cirs_ra, time):
...
+    """
+    Convert from astropy altaz convention to UVBeam az/za convention.
+
+    Args:
+        zenith_angle: in radians
+        azimuth: in radians in uvbeam convention: North of East(East=0, North=90)
+
+    Returns:
+        altitude in radians
+        azimuth in radians in astropy convention: East of North (N=0, E=90 degrees)
```
Comment (dannyjacobs) [inline pyuvsim/utils.py]: Notes in degrees, units in radians.

Returns:
        altitude in radians
        azimuth in radians in astropy convention: East of North (N=0, E=pi/2 degrees)

File: pyuvsim/analyticbeam.py (hunk 1 above)
Comment (bhazelton) [reply pyuvsim/analyticbeam.py]: I added a full docstring.

File: pyuvsim/source.py (hunk 2 above)
Comment (bhazelton) [reply pyuvsim/source.py]: I think we've consistently used radians, although some comments reference degrees which I will change.

Comment (bhazelton) [reply pyuvsim/source.py]: we do use degrees in some testing, but I think that's ok as they're always converted to radians before calling the functions.

File: pyuvsim/utils.py (hunk 3 above)
Comment (bhazelton) [reply pyuvsim/utils.py]: fixed.

**Comment by bhazelton** [comment]
Ok @dannyjacobs I think this is ready for review.

File: pyuvsim/source.py (hunk 2 above)
Comment (dannyjacobs) [reply pyuvsim/source.py]: yeah, thats what it looks like to me too.  Seems ok.

**Pull request merged**